src/features/content-explorer/item-list/ItemList.js

```diff
@@ -188,8 +189,31 @@ const ItemList = ({
         };
     }
 
+    const withAutoSizer = WrappedComponent => {
+        return props => {
+            return isResponsive ? (
+                <div style={{ flex: 1 }}>
+                    <AutoSizer>
+                        {({ width: w, height: h }) => <WrappedComponent {...props} width={w} height={h} />}
+                    </AutoSizer>
+                </div>
+            ) : (
+                <WrappedComponent {...props} />
+            );
+        };
+    };
+
+    TableComponent = withAutoSizer(TableComponent);
+
     return (
-        <div className={classNames('content-explorer-item-list table', className)}>
+        <div
+            className={classNames(
+                'content-explorer-item-list table',
+                // isResponsive ? 'content-explorer-item-list--isResponsive' : '',
+                className,
+                { 'bdl-ItemList--responsive': isResponsive },
```

Comment:
I've been thinking on how we want to handle this class name since it's likely `bdl-ItemList` would conflict with a similarly named component in our parent apps.

While it doesn't exactly following the convention, I think we should try `bdl-ContentExplorerItemList--responsive` with the intention of renaming this component to something else in the future.

Reply:
Updated to the new className.